### PR TITLE
Closes #1403: Add a private browsing icon [ci skip]

### LINKED ISSUE #1403: Add icon for private browsing
Currently, it's needed for showing private browsing tabs in the reference browser.

Photon icon: https://design.firefox.com/icons/viewer/#private

### PR DESCRIPTION
**Pull request by jonalmeida**

### CODE REVIEW

**Comment by jonalmeida** [comment]
> Merging #1429 into master will decrease coverage by 0.06%.

🤦‍♂️

**Comment by Amejia481** [comment]
🙃

File: components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
+    <path
+        android:pathData="M17.5,17c-2.1,0 -3.5,-2.5 -5.5,-2.5S8.4,17 6.5,17C3.9,17 2,14.6 2,10.4 2,7.8 2.8,7 6.1,7s4.3,1.4 5.9,1.4c1.6,0 2.6,-1.4 5.9,-1.4 3.3,0 4.1,0.8 4.1,3.4 0,4.2 -1.9,6.6 -4.5,6.6zM7.7,10.2c-2,0.1 -2.9,1.3 -2.9,1.6 0,0.3 1.3,1.1 2.7,1.1 1.3,0 2.9,-0.5 2.9,-0.9 0,-0.5 -0.8,-1.9 -2.7,-1.8zM16.3,10.2c-1.9,-0.1 -2.7,1.3 -2.7,1.8 0,0.4 1.5,0.9 2.9,0.9s2.7,-0.8 2.7,-1.1c-0.1,-0.3 -0.9,-1.5 -2.9,-1.6z"
+        android:fillColor="#ffffff"
+        android:fillAlpha=".8"/>
```
Comment (pocmo) [inline components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml]: Please land this without alpha!

File: components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (pocmo) [inline components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml]: Is this from the Photon icon pack?
* https://github.com/FirefoxUX/photon-icons/tree/master/icons/android
* https://design.firefox.com/icons/viewer/

File: components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="16"
+    android:viewportHeight="16">
```
Comment (pocmo) [inline components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml]: All other icons seem to use a viewport width/height of 24. Is this intentional?

File: components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml (hunk 2 above)
Comment (jonalmeida) [reply components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml]: Yes it is.

File: components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml (hunk 3 above)
Comment (jonalmeida) [reply components/ui/icons/src/main/res/drawable/mozac_ic_private_browsing.xml]: Interesting, I see some with 16 as well, but I'll change it to 24 if that's our standard.

**Pull request merged**